data/json/items/gun/762x25.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "id": "ppsh",
+    "type": "GUN",
+    "name": "PPSh-41",
+    "description": "The Soviet made PPSh-41, chambered in 7.62 Tokarev, provides a relatively large ammunition capacity coupled with low recoil and decent accuracy.",
+    "weight": 3630,
```

Comment:
In that case there's no need for a further change. Be aware this is usually a necessary step however when migrating items